assessment_software/src/app/app.component.html

```diff
@@ -1,4 +1,5 @@
 <p>Hello World</p>
+<router-outlet></router-outlet>
```

Comment:
This should be the only `<router-outlet></router-outlet>` in the whole angular project for our case. This router-outlet will be substituted with the matching page according to the URL. 

Remove other `<router-outlet></router-outlet>` from other pages.